### ISSUE TITLE
Bootstrap collapsed page menus and keyboard focus

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1388]
When viewed on small screens, the Bootstrap project page menu collapses, and page links are hidden until the menu button is clicked. The page links can still be tabbed to and selected when the menu is collapsed but no keyboard focus indicator is shown as the links are visually hidden. This fails [WCAG 2.4.7 (Focus Visible)](https://www.w3.org/WAI/WCAG22/quickref/?showtechniques=144%2C1410%2C211%2C222%2C247#focus-visible).

This issue is noted on the 3.14 accessibility statement
